conf/plugin-templates/common/dfs-storage-plugin.template

```diff
@@ -20,6 +20,30 @@
         "writable" : true,
         "defaultInputFormat" : "parquet"
       },
+      "drilltesttpchsf1json": {
```

Comment:
There are already existing workspaces for this data. Can we reuse - or - cleanup?